saas_app/internal_ops/forms.py

```diff
@@ -408,21 +408,21 @@ def __init__(self, *args, **kwargs):
             Field(
                 "manager",
                 readonly=True,
-                style="color:black; background-color:#e9ecef; opacity:1;font-weight: inherit;font-color: inherit;",
+                style="color:black; height:auto; ackground-color:#e9ecef; opacity:1;font-weight: inherit;font-color: inherit;",
```

Comment:
I think the background's `b` got eaten up in the change! :D

Reply:
Thank you for catching this - I totally did not see it!